.github/workflows/ci.yml

```diff
@@ -25,3 +25,19 @@ jobs:
           bundler-cache: true
 
       - run: bundle exec rake
+
+  rubocop:
+    name: Run rubocop
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: rm Gemfile.lock
```

Comment:
not a fan of this because it will bump rubocop and then result in new cops running/failing

Reply:
Yep, not sure what's desired policy. Just to have locked version and update it manually time to time?